plugins/omkafka/omkafka.c

```diff
@@ -581,8 +622,8 @@ writeKafka(instanceData *const pData, uchar *const msg,
 	pthread_rwlock_t *dynTopicLock = NULL;
 	failedmsg_entry* fmsgEntry;
 	int topic_mut_locked = 0;
-#if RD_KAFKA_VERSION >= 0x00090400
```

Comment:
Is this really correct? The #if seems to guard the kafka type, doen'st make so much sense to me to guard the int64_t. Or am I overlooking something?

Reply:
I only pulled rd_kafka_resp_err_t msg_kafka_response; from under #if as I can use it for both librdkafka version types. int64_t ttMsgTimestamp was also already there and is being used to pass timestamp of resubmit to rd_kafka_producev(). Interface for older version looks different, which is probably why it was conditionally declared for newer versions.